borg/key.py

```diff
@@ -126,28 +126,29 @@ class AESKeyBase(KeyBase):
     def id_hash(self, data):
         """Return HMAC hash using the "id" HMAC key
         """
-        return HMAC(self.id_key, data, sha256).digest()
+        return hmac_sha256(self.id_key, data)
```

Comment:
Seem like less flexible / harder to change than before.

One of my not-yet-merged changes in experimental branch was to make the hash/hmac flexible.
E.g. sha512-256 is faster on 64bit than sha256.